installation/sql/mysql/joomla.sql

```diff
@@ -24,7 +24,7 @@ CREATE TABLE IF NOT EXISTS `#__assets` (
   UNIQUE KEY `idx_asset_name` (`name`),
   KEY `idx_lft_rgt` (`lft`,`rgt`),
   KEY `idx_parent_id` (`parent_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+) ENGINE=MyIsam DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
```

Comment:
I would suggest to don't change this in your PR. If you really think that this should be a change in schema then send it to joomla-cms core and discuss there. but after your midterm evolution as I would like to see you more focus on current project. You can mark it as a todo.